CHANGELOG.md

```diff
@@ -3,6 +3,118 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.19.0
+
+_Oct 4, 2024_
+
+We'd like to offer a big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:
+
+- ✅ Automatic parents and children selection in the Data Grid Premium component
+- 💫 Support `minHeight` and `maxHeight` on flex parent container for the Data Grid component
+- 🎁 Export `publicAPI` form `useTreeItem2Utils` in the TreeView component
+- 🌍 Improve Bulgarian (bg-BG), Croatian (hr-HR), French (fr-FR), German (de-DE), Japanese (ja-JP) and Vietnamese (vi-VN) locales and add Portuguese (pt-PT) locale on the Data Grid component
```

Comment:
Nitpick: separate 
```suggestion
- 🌍 Add Portuguese (pt-PT) locale on the Data Grid component
- 🌍 Improve Bulgarian (bg-BG), Croatian (hr-HR), French (fr-FR), German (de-DE), Japanese (ja-JP), and Vietnamese (vi-VN) locales on the Data Grid component
```

Reply:
I structured it this way because you have similar description for the Pickers